package.json

```
@@ -1,11 +1,12 @@
 {
   "name": "@wdio/testrail-reporter",
-  "version": "0.3.0",
-  "description": "Create a run on testrail and the update the test cases results",
+  "version": "0.4.0",
```

Our release pipeline takes care of this
```suggestion
  "version": "0.3.0",
```